Lib/argparse.py

```diff
@@ -1709,14 +1709,7 @@ def _remove_action(self, action):
         self._group_actions.remove(action)
 
     def add_argument_group(self, *args, **kwargs):
-        import warnings
-        warnings.warn(
-            "Nesting argument groups is deprecated.",
-            category=DeprecationWarning,
-            stacklevel=2
-        )
-        return super().add_argument_group(*args, **kwargs)
-
+        raise ValueError('nested argument groups are not supported')
```

Comment:
I *think* this is the right error type but let me know if there's a more appropriate type.